bin/add_celltypes_to_sce.R

```diff
@@ -250,6 +267,47 @@ if (!is.null(opt$cellassign_predictions)) {
     }
     metadata(sce)$cellassign_reference_organs <- cellassign_organs
   }
+} else {
+  has_cellassign <- FALSE # use to decide if we should assign consensus labels
+}
+
+# assign consensus cell type labels
+if (has_singler & has_cellassign) {
```

Comment:
For clarity that there will only be one value.
```suggestion
if (has_singler && has_cellassign) {
```